bedrock/press/templates/press/press-inquiry.html

```diff
@@ -1,106 +1,88 @@
-{% extends "base-resp.html" %}
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-article.html" %}
 
 {% block page_title %}{{_('Press Inquiry Form')}}{% endblock %}
-{% block body_class %}sand{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('press') }}
+{{ css_bundle('press') }}
 {% endblock %}
 
-{% block content %}
-
-<main role="main" id="main-content">
-  <header class="main-header">
-    <h1 class="title-shadow-box">{{_('Press Inquiry')}}</h1>
-  </header>
-
-  <section class="container">
+{% block side_nav %}
+  {% include 'press/includes/resources.html' %}
+{% endblock %}
 
-    {% if not form_success %}
-      <h2 class="secondary-title">{{ _('Please complete and submit the form below with your press inquiry.') }}</h2>
+{% block article %}
+<header>
+  <h1 class="mzp-c-article-title">{{_('Press Inquiry')}}</h1>
+</header>
+{% if not form_success %}
+{% if form.errors %}
+<aside class="mzp-c-notification-bar mzp-t-error">
+  <p>
+    <strong>{{ _('An error has occurred with your submission. Please review your information below and try again.') }}</strong>
+  </p>
+</aside>
+{% endif %}
+
+{% if not form_success %}
+  <h2>{{ _('Please complete and submit the form below with your press inquiry.') }}</h2>
+{% endif %}
+
+<form name="mozilla-contact-us" id="press_inquiry" class="press-inquiry" action="{{ url('press.press-inquiry') }}" method="post">
+  {{ csrf() }}
+  <fieldset>
+    <label for="id_jobtitle">{{ _('Job Title') }}</label>
+    {% if form.jobtitle.errors %}
+    <strong class="error-msg">{{_('Please enter your job title.')}}</p>
```

Comment:
Mismatching tags here
```suggestion
    <strong class="error-msg">{{_('Please enter your job title.')}}</strong>
```